cmd/cosign/cli/verify/verify.go

```diff
@@ -141,6 +141,17 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 		NewBundleFormat:              c.NewBundleFormat,
 	}
 
+	// Check to see if we are using the new bundle format or not
+	if !c.LocalImage {
```

Comment:
Just to confirm, this should handle backwards compatibility for verification of either the new or old bundle format?